src/App.jsx

```diff
@@ -14,70 +16,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
+export const App = () => {
+  const [good, setGood] = useState('Jam');
+  const noGoodSelectedBlock = (
     <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+  );
 
+  const goodSelectedBlock = (
     <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+      {good}
+      {' is selected'}
 
       <button
         data-cy="ClearButton"
         type="button"
         className="delete ml-3"
+        onClick={() => setGood('')}
```

Comment:
```suggestion
        onClick={clearGood}
```

It's better to create a separate function as mentioned by the previous mentor.